doc/ci_admin_guide/Update_Jenkins_slaves.md

```diff
@@ -0,0 +1,108 @@
+# Updating Jenkins slaves
+
+## Rolling out new slaves
+
+TODO
+- update templates
+- create templates
+- spawn new slaves
+- decomission old slaves
+- tags
+- how to test if it worked safely
+
+## Updating existing slaves
+
+Some minor maintenance work does not require rollout of new slaves. An example might be an upgrade of
+some library or other dependency. In this case, there is no need to update the underlying slave template.
```

Comment:
> In this case, there is no need to update the underlying slave template.

That is not true. Following described procedure results in updated running cluster, but new slaves or replaced slaves would have different configuration.

I recommend noting that difference here.